ipycytoscape/cytoscape.py

```diff
@@ -814,12 +838,35 @@ class CytoscapeWidget(DOMWidget):
 
     graph = Instance(Graph, args=tuple()).tag(sync=True, **widget_serialization)
 
-    def __init__(self, **kwargs):
+    def __init__(self, graph=None, **kwargs):
+        """
+        Initializes the graph widget.
+
+        Parameters
+        ----------
+        graph: string or dict or pandas DataFrame object or networkx Graph
+               object or neo4j Graph object or Graph object, optional (defaults
+               to None)
+               Checked to be of one of the declared types, and graph object
+               corresponding to it is added as a Graph attribute to the
+               CytoscapeWidget DOM.
```

Comment:
```suggestion
               The graph to initialize with. Equivalent to calling the appropriate ``CytoscapeWidget.graph.add_graph_from_` method.
```

Is this clearer? We maybe should stay away from using words like `DOM` as I suspect that isn't a super clear term to most python users.